source/documentation/tools/create-a-derived-table/project-structure.md

```diff
@@ -1,30 +1,382 @@
+:::info
+
+## 3-intermediate
+
+Once we’ve got our atoms ready to work with, we’ll set about bringing them together into more intricate, connected molecular shapes. The intermediate layer is where these molecules live, creating varied forms with specific purposes on the way towards the more complex proteins and cells we’ll use to breathe life into our data products.
+
+### Intermediate: Files and folders
+
+Let’s take a look at the intermediate layer of our project to understand the purpose of this stage more concretely.
+
+```shell
+models/intermediate
+└── courts
+    └── courts_intermediate
+        ├── courts_intermidiate_properties.yml
+        └── int_int_xhibit_and_common_platform_receipts_unioned.sql
+```
+
+- **Folders**
+  - ✅ **Subdirectories based on business groupings.** Much like the staging layer, we’ll house this layer of models inside their own `intermediate` database. Unlike the staging layer, here we shift towards being business-conformed, splitting our models up into subdirectories not by their source system, but by their area of business concern.
+- **File names**
+  - `✅ int_[entity]s_[verb]s.sql` - the variety of transformations that can happen inside of the intermediate layer makes it harder to dictate strictly how to name them. The best guiding principle is to think about _verbs_ (e.g. `pivoted`, `aggregated_to_user`, `joined`, `fanned_out_by_quantity`, `funnel_created`, etc.) in the intermediate layer. In our example project, we use an intermediate model to pivot payments out to the order grain, so we name our model `int_payments_pivoted_to_orders`. It’s easy for anybody to quickly understand what’s happening in that model, even if they don’t know [SQL](https://mode.com/sql-tutorial/). That clarity is worth the long file name. It’s important to note that we’ve dropped the double underscores at this layer. In moving towards business-conformed concepts, we no longer need to separate a system and an entity and simply reference the unified entity if possible. In cases where you need intermediate models to operate at the source system level (e.g. `int_shopify__orders_summed`, `int_core__orders_summed` which you would later union), you’d preserve the double underscores. Some people like to separate the entity and verbs with double underscores as well. That’s a matter of preference, but in our experience, there is often an intrinsic connection between entities and verbs in this layer that make that difficult to maintain.
```

Comment:
CaDeT requires us to include the database in all model names:

- `✅ [datebase_name]__int_[entity]s_[verb]s.sql`

Reply:
or if `int_[entity]s` is the database name then `int_[entity]s__[verb]s.sql`

Reply:
Between this section and the section on line 100, I'm confused about double underscores. Can we clarify when CaDeT requires it to distinguish between database and model?

Reply:
I have removed this section as it confuses things and we cannot use double underscores in our file names:

  It’s important to note that we’ve dropped the double underscores at this layer. In moving towards business-conformed concepts, we no longer need to separate a system and an entity and simply reference the unified entity if possible. In cases where you need intermediate models to operate at the source system level (e.g. `int_shopify__orders_summed`, `int_core__orders_summed` which you would later union), you’d preserve the double underscores. Some people like to separate the entity and verbs with double underscores as well. That’s a matter of preference, but in our experience, there is often an intrinsic connection between entities and verbs in this layer that make that difficult to maintain.
  
 I have added you abopve comments @benwaterfield

Reply:
thank you - i think that extra detail very much helps here